app/models/rr_pending_change.rb

```diff
@@ -14,10 +14,7 @@ def self.last_id
   class << self; alias_method :backlog, :count; end
 
   def self.backlog_details
-    counts = all(
-      :select => "change_table, COUNT(id) AS count_all",
-      :group  => "change_table"
-    )
-    counts.each_with_object({}) { |c, h| h[c.change_table] = c.count_all.to_i }
+    counts = select("change_table, COUNT(id) AS count_all").group("change_table").order('count_all DESC').limit(20).to_a
+    counts.each_with_object([]) { |c, a| a << [c.change_table, c.count_all.to_i] }
```

Comment:
While we're in here, let's get right of the `each_with_object([])`, since that's an anti-pattern.  In addition, if you have unique change_tables, this would read nicer as a Hash (particularly when inspected).  So maybe something like

```ruby
# Remove the to_a from the previous line as it's redundant, then
counts.collect { |c| [c.change_table, c.count_all.to_i] }.to_h
```